WCFDataService/Client/System/Data/Services/Client/ClientEdmModel.cs

```diff
@@ -443,7 +447,8 @@ private EdmTypeCacheValue GetOrCreateEdmTypeInternal(IEdmStructuredType edmBaseT
                             List<IEdmProperty> loadedProperties = new List<IEdmProperty>();
                             foreach (PropertyInfo property in ClientTypeUtil.GetPropertiesOnType(type, /*declaredOnly*/edmBaseType != null).OrderBy(p => p.Name))
                             {
-                                IEdmProperty edmProperty = this.CreateEdmProperty(complexType, property);
+                                FieldInfo backingField = FindBackingField(property, this.ProxyBackingFieldNamingConvention);
```

Comment:
Sorry, I do not understand. Since for complex, it is written like: _private global::System.Collections.ObjectModel.ObservableCollection<ContactDetails> _BackupContactInfo = new global::System.Collections.ObjectModel.ObservableCollection<ContactDetails>();_. Same with fields. How does it be lazy constructed? Won't it be initialized immediately?